webapp/packages/core-settings/src/createSettingsAliasResolver.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import '@testing-library/jest-dom';
+
+import { SyncExecutor } from '@cloudbeaver/core-executor';
+
+import { expectDeprecatedSettingMessage } from './__custom_mocks__/expectDeprecatedSettingMessage';
+import { createSettingsAliasResolver } from './createSettingsAliasResolver';
+import type { ISettingsSource } from './ISettingsSource';
+
+const deprecatedSettings = {
+  deprecated: 'deprecatedValue',
+};
+
+const newSettings = {
+  ...deprecatedSettings,
+  value: 'value',
+};
+
+function createSource(settings: Record<any, any>): ISettingsSource {
```

Comment:
no probably, we will not reuse it, it's only for testing this current feature
anyway in general to move something to separate reusable module, you need to have at least two cases where you can reuse it. otherwise it has no sense.